### PR TITLE
Tweak timing for highlight animation

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Tweak timing of the hightlight animation after feedback from design.

- `TextField`: update text color while the animation is running.

## Justify why they are needed

Design feedback.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1739** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1739" target="_blank">Graphite</a> 
    * **PR #1741** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1741" target="_blank">Graphite</a> 
      * **PR #1744** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1744" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1744?utm_source=stack-comment).

File: apps/store/src/utils/useHighlightAnimation.ts [hunk 1]
```diff
@@ -22,14 +18,23 @@ export const useHighlightAnimation = () => {
...
       onAnimationComplete: () => setIsInteractive(false),
+      'data-highlight': isInteractive,
     },
   } as const
 }
 
+const ANIMATION_VARIANTS: Variants = {
+  [AnimationState.Active]: {
+    backgroundColor: theme.colors.green100,
+    transition: { ...theme.transitions.framer.easeInOutCubic, duration: 0.4 },
```
Comment (gustaveen) [inline apps/store/src/utils/useHighlightAnimation.ts:34]: Nice, didn't know we had in the theme 👌

**Review by gustaveen — APPROVED** [review]
👌 💅

File: apps/store/src/components/TextField/TextField.tsx [hunk 2]
```diff
@@ -116,13 +116,17 @@ const Label = styled.label({
     transform: `translate(calc(${theme.space.md} * 0.2), -0.8rem) scale(0.8)`,
   },
 
-  '&&[data-disabled=true]': {
-    color: theme.colors.textSecondary,
+  [`${LargeWrapper}[data-highlight=true] > &`]: {
```
[the remark this answers is not in the document]
Comment (robinandeer) [reply apps/store/src/components/TextField/TextField.tsx:119]: Mistake from me, I thought `SmallWrapper` would inherit these rules but it doesn't

Will fix in later PR!

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Job ID: sX2xr9E1lbYzBI4LtzFe.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1744)

**Pull request merged**